utils/economic-checks/fuzz/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "economic-checks-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+
+[dependencies.economic-checks]
+path = ".."
```

Comment:
I prefer specify deps in one style for one toml, so this should be included in common [dependencies] section